public/main.js

```diff
@@ -1,5 +1,38 @@
 const qrButton = document.getElementById('createqr')
 
+window.onload = qreator()
+
+function qreator() {
+  fetch('/qrcards')
+    .then(res => res.json())
+    .then(data => {
+      const $cardRow = document.getElementById('cardrow')
+      data.forEach((data) => {
+        $cardRow.appendChild(createCard(data))
+      })
+      if (data.length > 0) {
+        const $cardHeader = document.getElementById('cardheader')
+        $cardHeader.classList.remove('hidden')
+      }
+    })
+    .catch((err) => console.log(err))
+}
+
+function updateCard() {
```

Comment:
this function name is a little ambiguous with `updateQRData`. Maybe a better name?